account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -467,7 +474,7 @@ const generateSignedPermit = async <
       owner: account.address,
       spender: paymasterAddress,
       value: maxAmountToken,
-      nonce: BigInt(data),
+      nonce: nonce,
```

Comment:
(Nit) Can write more simply:
```suggestion
      nonce,
```